src/main/java/uk/gov/hmcts/darts/arm/service/impl/AbstractArmBatchProcessResponseFiles.java

```diff
@@ -462,56 +456,41 @@ private void processCreateRecordResponseFiles(List<CreateRecordFilenameProcessor
 
     private void readCreateRecordFile(BinaryData createRecordBinary, CreateRecordFilenameProcessor createRecordFilenameProcessor,
                                       ArmBatchResponses armBatchResponses) {
+        String createRecordFilenameAndPath = createRecordFilenameProcessor.getCreateRecordFilenameAndPath();
         if (nonNull(createRecordBinary)) {
-            Path jsonPath = null;
             try {
-                boolean appendUuidToWorkspace = true;
-                jsonPath = fileOperationService.saveBinaryDataToSpecifiedWorkspace(
-                    createRecordBinary,
-                    createRecordFilenameProcessor.getCreateRecordFilename(),
-                    armDataManagementConfiguration.getTempBlobWorkspace(),
-                    appendUuidToWorkspace
-                );
-
-                if (nonNull(jsonPath) && jsonPath.toFile().exists()) {
-                    logResponseFileContents(jsonPath);
-                    ArmResponseCreateRecord armResponseCreateRecord = getResponseCreateRecordOrDelete(jsonPath);
-                    UploadNewFileRecord uploadNewFileRecord = readInputJson(armResponseCreateRecord.getInput());
-                    if (nonNull(uploadNewFileRecord)) {
-                        if (StringUtils.isNotEmpty(uploadNewFileRecord.getRelationId())) {
-                            armBatchResponses.addResponseBatchData(Integer.valueOf(uploadNewFileRecord.getRelationId()),
-                                                                   armResponseCreateRecord, createRecordFilenameProcessor);
-                        } else {
-                            log.warn("Unable to get EOD id (relation id) from uploadNewFileRecord {} create record {}",
-                                     armResponseCreateRecord.getInput(), createRecordFilenameProcessor.getCreateRecordFilenameAndPath());
-                        }
+                log.info("Contents of ARM CR response file: {} - {}", createRecordFilenameAndPath, createRecordBinary);
+                ArmResponseCreateRecord armResponseCreateRecord = getResponseCreateRecordOrDelete(createRecordFilenameAndPath, createRecordBinary);
+                UploadNewFileRecord uploadNewFileRecord = readInputJson(armResponseCreateRecord.getInput());
+                if (nonNull(uploadNewFileRecord)) {
+                    if (StringUtils.isNotEmpty(uploadNewFileRecord.getRelationId())) {
+                        armBatchResponses.addResponseBatchData(Integer.valueOf(uploadNewFileRecord.getRelationId()),
+                                                               armResponseCreateRecord, createRecordFilenameProcessor);
                     } else {
-                        log.warn("Failed to obtain EOD id (relation id) from create record file  {}",
-                                 createRecordFilenameProcessor.getCreateRecordFilenameAndPath());
+                        log.warn("Unable to get EOD id (relation id) from uploadNewFileRecord {} create record {}",
```

Comment:
dont we delete the response file if EDO is not found?

Reply:
line 463 calls the method that either gets the json or deletes the file if the json cannot be read